test/e2e/cluster/stack.go

```diff
@@ -108,7 +110,15 @@ func (s *stack) deploy(ctx context.Context, test TestResources) (*resourcesStack
 	return result, nil
 }
 
-func (s *stack) prepareStackParameters(test TestResources) []types.Parameter {
+func (s *stack) prepareStackParameters(test TestResources, eks EKSConfig) []types.Parameter {
+	clusterRoleSP := defaultClusterRoleSP
+	if eks.ClusterRoleSP != "" {
+		clusterRoleSP = eks.ClusterRoleSP
+	}
+	podIdentitySP := defaultPodIdentitySP
+	if eks.PodIdentitySP != "" {
+		podIdentitySP = eks.PodIdentitySP
+	}
```

Comment:
wdyt about setting these defaults where we read the config? that way the struct has the right values all the way in